widget/label.go

```diff
@@ -114,10 +103,31 @@ func (l *Label) Unbind() {
 	}
 
 	l.textSource.RemoveListener(l.textListener)
-	l.textListener = nil
 	l.textSource = nil
 }
 
+func (l *Label) createListener() {
+	if l.textListener != nil {
+		return
+	}
+
+	l.textListener = binding.NewDataListener(func() {
+		if l.textSource == nil {
```

Comment:
It seems that this part should be written as:
```go
src := l.textSource
if src == nil {
    return
}
val, err := src.Get()
```
To nullify any chance to have a nil textSource when we actually call `Get()`.
This is because it seems that there is a little probability that textSource is nil when Go (internally) is converting the interface to its actual type? (I am supposing this, as that could be the only reason to explain this). Take a look this code:
https://play.golang.org/p/bqEsHGUYdfw